suite-common/graph/src/constants.ts

```diff
@@ -9,10 +9,14 @@ export const LOCAL_BALANCE_HISTORY_COINS = [
     'pol',
     'bnb',
     'xrp',
-    'sol',
 ] satisfies Array<NetworkSymbol>;
 export type LocalBalanceHistoryCoin = (typeof LOCAL_BALANCE_HISTORY_COINS)[number];
 
 export const isLocalBalanceHistoryCoin = (
     symbol: NetworkSymbol,
 ): symbol is LocalBalanceHistoryCoin => isArrayMember(symbol, LOCAL_BALANCE_HISTORY_COINS);
+
+export const IGNORED_BALANCE_HISTORY_COINS = ['sol'] satisfies Array<NetworkSymbol>;
```

Comment:
maybe add some comments why we need to ignore some coins

Reply:
added in 14995a1a5b